static/src/javascripts/projects/commercial/modules/header-bidding/utils.ts

```diff
@@ -142,12 +140,11 @@ export const shouldIncludeXaxis = (): boolean => isInUk();
 
 export const shouldIncludeImproveDigital = (): boolean => isInUk() || isInRow();
 export const shouldIncludeImproveDigitalSkin = (): boolean => {
-	if (!(isInUk() || isInRow())) return false;
-	return (
-		window.guardian.config.page.isFront &&
-		getBreakpointKey() === 'D' &&
-		isInABTestSynchronous(improveSkins)
-	);
+	return [
+		window.guardian.config.page.isFront,
+		isInUk(),
+		getBreakpointKey() === 'D',
+	].every(Boolean);
```

Comment:
I was curious about this pattern and ran a few tests in the console - it turns out that using `&&` to compare 3 elements is over 3 times faster than initialising an array and then running `Boolean` over each element. 
The `&&` operator took 3.57 seconds whereas `[...].every(Boolean)` took 11.1 seconds over 1 million iterations. If this code runs every time there's a Prebid auction it's probably worth using `&&`! 
What do you think?

Reply:
Ha, I was actually wondering about the performance of this. Thanks for checking! I’ll revert back.

Do you mind sharing the benchmark?

Reply:
Sorry, late reply! Here's the screenshot: 
![image](https://user-images.githubusercontent.com/57295823/127468817-7850cd13-932a-4b6d-b42d-6bbb451e4d1f.png)

Reply:
Deno is (way) faster, but there’s still a 8× speedup.

```ts
const iterations = 1_000_000;

function test(id: string, fun: () => void) {
  console.time(id);
  fun();
  console.timeEnd(id);
}

test("every(Boolean)", () => {
  for (let index = 0; index < iterations; index++) {
    const _ = [true, true, true].every(Boolean);
  }
});

test("straight conditions", () => {
  for (let index = 0; index < iterations; index++) {
    const _ = true && true && true;
  }
});

```